src/test/java/org/myrobotlab/service/AbstractServiceTest.java

```diff
@@ -1,49 +0,0 @@
-package org.myrobotlab.service;
-
-import org.junit.Test;
-import org.myrobotlab.framework.Service;
-import org.myrobotlab.logging.LoggingFactory;
-import org.junit.Ignore;
-
-/**
- * A base generic test class that can start a service, call a test method on it,
- * and then release the service The idea is you can subclass this and implement
- * the createService() and testService() methods to instrument a unit test for
- * any service in MRL.
- * 
- * @author kwatters
- *
- */
-@Ignore
-public abstract class AbstractServiceTest {
```

Comment:
Seems like having a base class for testing services specifically would be useful.  We have lots of services to test.  Having a base class for those tests would be desirable.

Reply:
Sure .. it should extend AbstractTest ..